ui/docs/Architecture.md

```diff
@@ -0,0 +1,96 @@
+# Architecture
+
+The UI for the starter application is implemented using React, as well as a set
+of other dependancies, such as Webpack for build and Jest as a test runner. The
+UI however is one part of the whole starter application.
+
+The following documentation will describe the topology of the starter 
+application when developing the UI, as well as when it is deployed as a
+part of the fully built starter application. In addition, it will describe
+the composition of React components used to build up the UI.
+
+## Topology summary
+
+### When deployed
+
+The starter application's topology when run by a user will resemble the 
+following:
+
+![Starter app topology](./ArchImages/Topology.jpg)
+
+When running the Jar file, a Vertx server will be started, which in turn will
+launch three verticles. One verticle will handle records being produced to
+Kafka, one verticle will handle records being consumed from Kafka, and finally
```

Comment:
This isn't quite right. The verticle that hosts the websocket also does the consuming.

Reply:
Shall update